inverse.go

```diff
@@ -184,16 +184,56 @@ func (i *InverseBloomFilter) WriteTo(stream io.Writer) (int64, error) {
 // have been written by WriteTo()) from an i/o stream. It returns the number
 // of bytes read.
 func (i *InverseBloomFilter) ReadFrom(stream io.Reader) (int64, error) {
+	decoded, capacity, size, err := i.decodeToArray(stream)
+	if err != nil {
+		return int64(0), err
+	}
+
+	// Create []*[]byte and point to each item in decoded
+	decodedWithPointers := make([]*[]byte, capacity)
+	for p := range decodedWithPointers {
+		if len(decoded[p]) == 0 {
+			decodedWithPointers[p] = nil
+		} else {
+			decodedWithPointers[p] = &decoded[p]
+		}
+	}
+
+	i.array = decodedWithPointers
+	i.capacity = uint(capacity)
+	return int64(size) + int64(2*binary.Size(uint64(0))), nil
+}
+
+// ImportElementsFrom reads a binary representation of InverseBloomFilter (such as might
+// have been written by WriteTo()) from an i/o stream. It returns the length of
+// the received elements.
+func (i *InverseBloomFilter) ImportElementsFrom(stream io.Reader) (int, error) {
```

Comment:
so what's the difference between `ReadFrom` and `ImportElementsFrom` ? we should probably be explicit in the comments, because right now they're the same